verify-scripts-json.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python
+
+import os
+from os import listdir
+import json
+from os.path import isfile, join
+
+
+dir = os.path.dirname(__file__)
+scriptlist = [f for f in listdir(os.path.join(dir,'scripts')) if isfile(join(os.path.join(dir,'scripts'), f))]
+missing = []
+missingrns = []
+tlslite = json.load(open(os.path.join(dir,'tests/tlslite-ng.json')))
+tlslitern = json.load(open(os.path.join(dir,'tests/tlslite-ng-random-subset.json')))
```

Comment:
do you need to check here? My understanding is that `tlsline-ng.json` would have everything, but I haven't really checked

Reply:
Indeed tlsline-ng.json is only used in travis but according to the issue description "Add script that runs in Travis that will verify that all scripts in scripts/are mentioned at least once in every json file.", so I am not really sure how is it intended to work exactly.